otisweb/templates/generic_form.html

```diff
@@ -6,28 +6,32 @@
   {% csrf_token %}
   {% if form.non_field_errors %}
     <div class="alert alert-danger" role="alert">
+      <strong>{{ "Error"|title }}</strong>
```

Comment:
why is the `|title` filter necessary here?

Reply:
maybe better use `<h3 class="alert-heading"></h3>` instead of `<strong>` as well.

Reply:
It was so it looks similar to the non field error style